course/en/chapter13/grpo_finetune.ipynb

```diff
@@ -0,0 +1,175 @@
+{
```

Comment:
<p>Maybe add one-liner headings/ description above each cell - otherwise it looks a bit too bare IMO</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>